examples/move/closed_loop/examples/regulated_coin.move

```diff
@@ -0,0 +1,318 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// This example demonstrates how to use Closed Loop to create a regulated coin
+/// that follows different regulatory requirements for actions:
+///
+/// 1. A new Token can only be minted by admin (out of scope)
+/// 2. Tokens can only be transferred between KYC-d (approved) addresses
+/// 3. A single transfer can't exceed 3000.00 REG
+/// 4. A single "withdraw" operation `to_coin` can't exceed 1000.00 REG
+/// 5. All actions are regulated by a denylist rule
+///
+/// With this set of rules new accounts can either be created by admin (with a
+/// mint and transfer operation) or if the account is KYC-d it can be created
+/// with a transfer operation from an existing account. Similarly, an account
+/// that has "Coin<REG>" can only convert it to `Token<REG>` if it's KYC-d.
+///
+/// Notes:
+///
+/// - best analogy for regulated account (Token) and unregulated account (Coin)
+/// is a Bank account and Cash. Bank account is regulated and requires KYC to
+/// open, Cash is unregulated and can be used by anyone and passed freely.
+/// However should someone decide to put Cash into a Bank account, the Bank will
+/// require KYC.
+///
+/// - KYC in this example is represented by an allowlist rule
+module examples::regulated_coin {
```

Comment:
Like this--ends up being super clean once someone spots you the rules.

I do think it would be useful to have a separate, simpler example that has the denylist only as a template for stablecoin issuers to start from.

Reply:
Just added `simple_token`.